test/parallel/test-util-types.js

```diff
@@ -0,0 +1,99 @@
+/*global SharedArrayBuffer*/
+'use strict';
```

Comment:
There's already https://github.com/nodejs/node/blob/master/test/parallel/test-types.js. You might want to look into combining these two files.

Reply:
@TimothyGu I’ve just copied the contents into a block in this file, let me know if that doesn’t sound good to you. The tests have somewhat different structures, but I think that’s fine?